portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,4 +49,14 @@ private String convertToJsonUsingGson(ArrayList toConvert) {
     String json = gson.toJson(toConvert);
     return json;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = request.getParameter("comment-input");
```

Comment:
Can you create a string for this and add a comment to what it means?

Reply:
I meant a private static final string.

Reply:
Done!